Assets/Scripts/KeyLocationController.cs

```diff
@@ -65,4 +84,31 @@ private IEnumerator GetPlayers()
             players.AddRange(GameObject.FindGameObjectsWithTag("Player"));
         }
     }
+
+    public void LoseHealth() //Run this when enemies are killed and reach location
+    {
+        health -= 1;
+    }
+
+    private void DestroyLocation()
+    {
+        sphere.transform.localScale -= new Vector3(shrinkValue, shrinkValue, shrinkValue) * Time.deltaTime;
```

Comment:
Doesn't Vector3 have a constructor with 1 value to pass and would look something like `new Vector3(shrinkValue)`?

Reply:
Nope, I'm wrong